### PR TITLE
fix: add srishti666

### LINKED ISSUE #1768: [PROFILE ADDITION] <SRISHTI MANNA>
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by srishti666**
resolve #1768

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please create a valid GitHub issue as explained in the [readme](https://github.com/EddieHubCommunity/awesome-github-profiles#how-can-i-add-my-github-profile-%EF%B8%8F%EF%B8%8F).
Let us know if you need any help with that. Thanks. 🙂

**Comment by srishti666** [comment]
I have accidentally written the wrong issue number , my correct issue number is #1768

File: profiles/srishti666.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/85846340/226453167-a4273fc1-f72e-4edd-a5e7-e5842b74c9df.png",
+  "issueId": 1759,
```
Comment (schmelto) [inline profiles/srishti666.json:3]:
```suggestion
  "issueId": 1768,
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**